scripts/Tools/compare_test_results

```diff
@@ -87,8 +95,8 @@ def _main_func(description):
     baseline_name, baseline_root, test_root, compiler, test_id, compare_tests = \
         parse_command_line(sys.argv, description)
 
-    success = compare_test_results(baseline_name, baseline_root, test_root, compiler, test_id, compare_tests)
-    sys.exit(0 if success else 1)
+    compare_test_results(baseline_name, baseline_root, test_root, compiler, test_id, compare_tests)
```

Comment:
Okay, thanks - I'll use TESTS_FAILED_ERR_CODE. I'll return that error code if there were any FAIL results, though will still return an exit code of 0 if there were some SKIP results. (Rationale: I figure it's not the responsibility of this script to determine if tests didn't pass - its responsibility is just to determine if there are any baseline comparison differences among tests that passed.)